.prettierignore

```diff
@@ -0,0 +1,5 @@
+dist
+node_modules
+package.json
+package-lock.json
+*.html
```

Comment:
Why do we ignore JSON and HTML, not an issue, just curious? 😄

Reply:
I guess JSON files can be removed, but they are still not included in the format script? Would you prefer to add them there?

As for HTML, it was excluded due to the default `printWidth` being 80, which I think is a bit narrow for them. Moreover, I don't think any changes will be needed there in the near future, and I have doubts that we even require these examples within the library.

Reply:
I mean if we run a formatter anyway, what's the harm in just letting it format all our files (except node_modules, etc of course).

Reply:
Up to you, I'm not too bothered about this 👍🏻